### PR TITLE
Adapt to utils moved to core, #49

### PR DESCRIPTION
**Pull request by kba**
This is the companion PR to https://github.com/OCR-D/core/pull/268

### CODE REVIEW

File: ocrd_tesserocr/common.py [hunk 1]
```diff
@@ -1,151 +1,24 @@
...
-    Return a numpy array of the resulting polygon.
-    """
-    angle = np.deg2rad(angle)
-    cos = np.cos(angle)
-    sin = np.sin(angle)
-    # active rotation:  [[cos, -sin], [sin, cos]]
-    # passive rotation: [[cos, sin], [-sin, cos]] (inverse)
-    return orig + np.dot(polygon - orig, np.array([[cos, sin], [-sin, cos]]).transpose())
 
 # to be refactored into core (as method of ocrd.workspace.Workspace):
```
Comment (bertsky) [inline ocrd_tesserocr/common.py]: Comment can be removed, too.

**Comment by bertsky** [comment]
Strange – my 2 comments did not end up in the context of the diffs I placed them. I will hide them and re-review.

**Comment by kba** [comment]
> Really? (I thought that seeing those FIXMEs in the report one by one is a major reason for running pylint?)

Was an oversight, just disabled to concentrate on code changes, will fix.

> Could you delete the file altogether (to avoid confusion in the future)?

Which file do you mean? pylintrc?

**Comment by bertsky** [comment]
> > Could you delete the file altogether (to avoid confusion in the future)?
> 
> Which file do you mean? pylintrc?

No, `common.py`

**Pull request merged**